### ISSUE TITLE
runtime error about ./config.json

### ISSUE THREAD

**Issue by cdedwards** [DustinBrett/daedalOS#44]
I keep getting this error from ArchLinux
[edwards@graphics daedalOS]$ uname -a
Linux graphics 5.16.2-arch1-1 #1 SMP PREEMPT Thu, 20 Jan 2022 16:18:29 +0000 x86_64 GNU/Linux
[edwards@graphics daedalOS]$ 

[edwards@graphics daedalOS]$ yarn dev
yarn run v1.22.17
$ next dev
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Disabled SWC as replacement for Babel because of custom Babel configuration ".babelrc" https://nextjs.org/docs/messages/swc-disabled
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './package.json' is not defined by "exports" in /home/edwards/git/daedalOS/node_modules/react/package.json
    at new NodeError (node:internal/errors:371:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:429:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:683:3)
    at resolveExports (node:internal/modules/cjs/loader:482:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.mod._resolveFilename (/home/edwards/git/daedalOS/node_modules/next/dist/build/webpack/require-hook.js:179:28)
    at Function.resolve (node:internal/modules/cjs/helpers:108:19)
    at getPackagePath (/home/edwards/git/daedalOS/node_modules/next/dist/build/webpack-config.js:548:41)
    at Object.getBaseWebpackConfig [as default] (/home/edwards/git/daedalOS/node_modules/next/dist/build/webpack-config.js:564:9) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[edwards@graphics daedalOS]$

**Comment by DustinBrett** [comment]
Thanks for pointing this out! Indeed there may be an issue running these commands with Linux as I have written all scripts on Windows with different path name requirements. I need to generalize my scripts which I plan to do quite soon when I am also building the Docker file everyone is requesting which will likely have me trying to get the exact same commands working under a Linux environment of some kind.

**Comment by DustinBrett** [comment]
As an alternative, I have added a Dockerfile along with instructions. If you have Docker you can run it without requiring yarn/node.

https://github.com/DustinBrett/daedalOS#docker

**Comment by DustinBrett** [comment]
@cdedwards I've looked into this further using ArchLinux in Docker and I think I've found a possible source for your issue. At least for me, when I setup ArchLinux I ran `packman -Sy nodejs`, but that installs the current v17 which is not LTS and actually doesn't work. It needs to be nodejs [LTS](https://nodejs.org/en/) (v16) which in this case I think is `pacman -Sy nodejs-lts-gallium`. Once I switched to that version it ran fine.
